### PR TITLE
upgrade deps

### PR DESCRIPTION
**Pull request by longlho**
Primarily upgrading cldr to 35 but currently blocked by https://github.com/eemeli/make-plural/pull/13 due to an issue in `kw` locale

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,23 +4,23 @@
...
   },
   "author": "Eric Ferraiuolo <edf@ericf.me>",
   "license": "BSD-3-Clause",
   "dependencies": {
-    "cldr-core": "^34.0.0",
-    "cldr-dates-full": "^34.0.0",
-    "cldr-numbers-full": "^34.0.0",
-    "glob": "^5.0.1",
-    "make-plural": "^2.1.3",
+    "cldr-core": "35",
```
Comment (redonkulus) [inline package.json]: Why use 35 versus ^?

Comment (longlho) [reply package.json]: it's the same really. I'll prob have to revert to 34 until the upstream CLDR issue for `kw` locale is resolved
https://github.com/unicode-org/cldr/pull/6

File: package.json [hunk 2]
```diff
@@ -4,23 +4,23 @@
...
-    "cldr-core": "^34.0.0",
-    "cldr-dates-full": "^34.0.0",
-    "cldr-numbers-full": "^34.0.0",
-    "glob": "^5.0.1",
-    "make-plural": "^2.1.3",
+    "cldr-core": "34",
+    "cldr-dates-full": "34",
+    "cldr-numbers-full": "34",
+    "glob": "7",
+    "make-plural": "4",
```
Comment (okuryu) [inline package.json]: Why did you choose not to use semver? Personally, I feel uncomfortable.

Comment (longlho) [reply package.json]: `34` is semver, it's the same as `^34.0.0`. I personally don't mind either way, just less chars. I can change to that if you want 😄

**Comment by longlho** [comment]
@okuryu @redonkulus build fixed 😄

**Comment by okuryu** [comment]
Looks good to me. I'll merge if there are no concern from @redonkulus.

**Pull request merged**